### ISSUE TITLE
[Feature] 请问关系图graph中，使用力导向图后，还有办法让node分布在特定区域范围吗？

### ISSUE THREAD

**Issue by luojichen** [apache/echarts#18487]
### What problem does this feature solve?

我的需求是，使用关系图，将所有的IP攻击关系图展示在graph内。 我使用了力导向图，确实可以自动布局，但是几个机房的内网IP和外网IP节点都会随机混乱的分布。我希望能够按照不同的IDC机房来分布我的 node节点（即内网被攻击的IP）。

譬如机房1的 IP， 我想要分布在左下角区域， 机房2的IP，我想要固定在右下角区域。  外网IP随机分布在上侧。

如果使用环形布局，显然更乱， 而使用坐标轴布局等于要重写一个布局算法。  唯一希望是力导向图，能够设定一个node的 基本分布范围， 来解决这个问题

### What does the proposed API look like?

？

**Comment by helgasoft** [comment]
> a way to distribute the nodes in a specific area?

yes, with [left](https://echarts.apache.org/en/option.html#series-graph.left), top, width and height.
[Demo code](https://echarts.apache.org/examples/en/editor.html?c=graph-force2&code=GYVwdgxgLglg9mABBATgUwIZTQOTgEzQGcAKCOcKASkQG8AoRRANwxUTAOMQF5EBtALoBuRomBx2JVuxi9EABmGI5AHmQUwUZTADUumgyZNOhIgDoADiCIALEkeMmMAWzQAuFYl2IA5L7EmAF8qUWCxdCgQFCRTYlEg-lBIWARkdCw0AFF8AHNiMk1qOjEZRDQ87j4hMJVgRELKXh4-ACZDQMRI6KR-fgUAGkQARkERMUSmCSkyuT4lL3VySh19DqcK_ItrOxJ-GCGSOR9hmgBSDUpBUImItCiY8sqiBPoy_CwMInkapMkG2byBZqRAADlWBhKTA-UC-Vhs9kcJi4RE8qEw2DwZiO3kQ7QGnU2xDRGWwOS2OJ87QmN0SAHo6YgMPh8F04HAXER6CgiFAAJ4AGzQ8kcRD5LgARnABZ5fPgYK4EPhfASmDBsC4AMr8oWeWgaAWSWUuDD5LQYXyIIKqxACjAStAyuiIOxwADunigKBAaCGljgRHV8DAsqgcEsKvECCgAHU0DBcrYoLKpQLlVb6Il6FAXJZ5GgILY2FALCBYALzBBDWA0CQeTq0Dcc5ZzGBXGg-L4UOyXK1fKIYV9-AB2QStlHw3bNm6DogjsdE77VcbTAFsLzzHSIdQATghNDEs_n5kXk_s_GbrfbQxg11E2dz-cLxdL5cr1dr9cFjdEl7bbk7bsOWGft6CPAA2Mc4m2BESGnZQwM-OdIJPZ4fhXf5pHXOZFC3dRwRUNZEEPJD-BQ08dnPP9rxUO96HocNUiQPhHDDaVYEsPVrTEIg0BQGBiUQWdzBNSwSGSaBgwadU0BcG98AAD3WYxukeJFjH5SwPD8XIUAwSxbBVTomDtPkKGTPxpggNAjKcJgMDAGATSYzxgAwAVeJtJxB08GSXHHMwvOMIVgAso5FMQC4AGYaAAKkQKKotxXwzlsuyw04xAAFksFscxgENSRwoUxBGRixB4sS5LUqCpg3RgfAoFsWVEpq4zEFseNEws3xWrSpwrO09SnAZGBHKgAAZDAzLLWVOBrSMGV8CAYBQCAQDtFB-rshldIwZh1T5TwFHMBQFGGIZ2qYdBrA84NPAAVge2rjCJCa0DAXImseh72u4uyni2XyNVQrYRP08TwEktISEbKEAa6e4enhhGXQoNbtLQfpBGS7aAdhFB8gsrHRlxl6nAFMa0G1b89UQerGuahKM1RoJaicEJOjZmlM2EIA)
![image](https://user-images.githubusercontent.com/13038071/230791678-0571bcfb-81d4-4df0-885d-9ff321f1c45d.png)
